### DISCUSSION TITLE
Blackboard integration

### DISCUSSION THREAD

**Discussion by chticer** [community/community#44066] [GitHub Education]
I am looking to see if there are any future plans to integrate Blackboard with GitHub classroom to import student rosters. I looked around GitHub resources and found this [GitHub Education Community Forum post](https://education.github.community/t/cannot-skip-lms-to-add-students/49914/4) from February 2020 saying that Blackboard support was coming in the future. I couldn't find any more recent updates on the old GitHub Education Community Forum or the new GitHub Education Discussions pages.

**Comment by samus-aran** [comment]
Hi @chticer 

Thanks for sharing and we appreciate your patience.

I have shared this with the internal team for your support of this feature. 

We don't officially support Blackboard yet but it seems that Blackboard supports LTI 1.3. As the Classroom roster integration also supports LTI 1.3 it may work with Blackboard as it currently is.  However we will recommend if you test the integration between Classroom and Blackboard that it would be at your own discretion.

**Reply by Pricer187** [reply]
Ok thank you

**Reply by Pricer187** [reply]
Ok thank you

**Comment by marc-hanheide** [comment]
Have you guys had any success integrating blackboard with GitHub classroom? We're looking to achieve the same and would benefit from some "success story".

**Comment by scollovati** [comment]
Hi @chticer, @samus-aran and @marc-hanheide 

I've completed testing and documentation for integrating Blackboard LMS with GitHub Classroom. The process is straightforward, and I've submitted a pull request (PR) to update the official documentation. You can find the PR, including a preview of the guide, at: https://github.com/github/docs/pull/35028

If you'd like to test the integration or review the documentation, please feel free to check out the PR. Your feedback would be appreciated.

**Reply by md-k-sarker** [reply]
Any update on this one?
